adoc/deployment-sysreqs.adoc

```diff
@@ -123,6 +123,10 @@ link:https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/hardware
 |External
 |Dex (OIDC Connect)
 
+|32001
```

Comment:
It seems incorrect formatting when preview by html mode. Please double confirm.

Reply:
Thanks. Finally, I know what's `.N+` meaning.